apps/storefront-e2e/src/support/page-fragments/category-navigation.fragment.ts

```diff
@@ -0,0 +1,5 @@
+export class CategoryNavigationFragment {
+  get = () => cy.get(`[uid='category-navigation']`);
```

Comment:
Usually, we call this method `getWrapper` to make it more readable, instead of `this.get()` which can be interpreted differently